index.js

```diff
@@ -457,16 +457,17 @@ module.exports = {
           return;
         }
       }
+      var skipThis = _.includes(self.excludeTypes, page.type);
       var url;
-      if (self.format === 'text') {
+      if (!skipThis && self.format === 'text') {
         if (self.indent) {
           var i;
           for (i = 0; (i < page.level); i++) {
             self.write(locale, '  ');
           }
           self.write(locale, page._url + '\n');
         }
-      } else {
+      } else if (!skipThis) {
```

Comment:
Why not an outer if to avoid duplication of the clause?

Reply:
My understanding is that this would be more performant than nested conditionals, particularly by only making the `_.includes` check once.